example_env.sh

```diff
@@ -4,4 +4,7 @@ DATA_SERVER_LOG_LVL=
 DATA_SERVER_PUBLIC_HOST=
 DATA_SERVER_PORT=
 FIRST_SUPERUSER=
-FIRST_SUPERUSER_PASSWORD=
\ No newline at end of file
+FIRST_SUPERUSER_PASSWORD=
+ENABLE_PRUNER=
+PRUNER_DIRECTORY=
```

Comment:
Do you think it would be better to allow more than one directory along with data retention period? Thoughts?

Reply:
yeah , so if they leave the PRUNER DIRECTORY var empty the pruner will run for all the folders of snappy , but in our case we want it to run only for the automated ci runs which get stored in perf-ci folder.